micronota/db/uniref.py

```diff
@@ -314,52 +220,55 @@ def prepare_metadata(in_fp, out_fp, append=True):
     The schema of the database file contains one table named `tigrfam` that
     has following columns:
 
-    1. ``ac``. TEXT. UniProtKB accession.
+    1. ``ac``. TEXT. UniProtKB primary accession.
 
-    2. ``dataset``. TEXT.
+    2. ``status``. INT. The index in ``_status``
 
-    3. ``taxon``. TEXT.
-
-    4. ``transfer``. INTEGER. Used as the boolean. ``1`` means the ``val``
-       should be transferred to the query sequences as its annotation;
-       ``0`` means not.
+    3. ``kingdom``. INT. The index in ``_kingdom``
```

Comment:
Probably should elaborate more on this (i.e. what do the indices correspond to.)